src/main/webapp/app/overview/overview.component.html

```diff
@@ -8,18 +8,43 @@ <h3> {{"arTeMiSApp.studentDashboard.exerciseTitle" | translate:{
 </div>
 <div class="row mb-3" *ngIf="nextRelevantExercise">
     <div class="col-12 pl-4 pr-4">
-        <jhi-course-exercise-row
-            class="pb-1"
-            [exercise]="nextRelevantExercise"
-            [course]="nextRelevantCourse"
-            [extendedLink]="true"
-        ></jhi-course-exercise-row>
+        <jhi-course-exercise-row class="pb-1" [exercise]="nextRelevantExercise" [course]="nextRelevantCourse"
+            [extendedLink]="true"></jhi-course-exercise-row>
     </div>
 </div>
-<div class="row mb-3">
-    <div class="col">
+<div class="row mb-3 align-items-center">
+    <div class="col-12 col-sm">
         <h3 jhiTranslate="arTeMiSApp.studentDashboard.title">Your current courses</h3>
     </div>
+    <div class="col-12 col-sm-auto" *ngIf="!showCourseSelection && !addedSuccessful">
```

Comment:
I am thinking if this should be its own component. Both template and controller seem to be fairly disconnected form the other functionality of the component. What do you think?

Reply:
I agree that this might make sense, however I would keep it simple for now, there is also some overhead in extracting this into its own component. We can still extract this functionality into its own component later if it makes sense.